pkg/abstract/model/endpoint_rotator_config_test.go

```diff
@@ -26,12 +26,15 @@ func TestRotatorConfig(t *testing.T) {
 }
 
 // some month utility
-var monthIds = map[time.Month]int{time.January: 0, time.February: 1, time.March: 2, time.April: 3, time.May: 4,
-	time.June: 5, time.July: 6, time.August: 7, time.September: 8, time.October: 9, time.November: 10, time.December: 11}
-var monthList = []time.Month{time.January, time.February, time.March, time.April, time.May,
-	time.June, time.July, time.August, time.September, time.October, time.November, time.December}
-var monthDayCount = []int{31, 28, 31, 30, 31, 30, 31, 31, 30, 31, 30, 31}
-var monthDayCountLeap = []int{31, 29, 31, 30, 31, 30, 31, 31, 30, 31, 30, 31}
+var monthIds = map[time.Month]int{
```

Comment:
i saw this change in other PR, probably will ended up with a conflict.

Reply:
I'll fix any conflicts